script/lib/transifex.py

```diff
@@ -499,6 +499,8 @@ def braveify(string_value):
             .replace('Brave Play', 'Google Play')
             .replace('Brave Safe', 'Google Safe')
             .replace('Sends URLs of some pages you visit to Brave', 'Sends URLs of some pages you visit to Google'))
+            .replace('Brave will save this password in your Brave Account', 'Brave will save this password in your Brave sync chain')
+            .replace('Passwords are saved in your Brave Account', 'Passwords are saved in your Brave sync chain')
```

Comment:
Should run npm run chromium_rebase_l10n to see the grd or grdp changes.
Maybe try Brave Account -> Brave sync chain instead to see if it produces sane results?

Reply:
Changing from `Brave Account` to `Brave sync chain` will be fine. Changed.
Should we run `chromium_rebase_l10n` for getting this change on local?
`push_l10n` only does this string replacement.
Previously, we should do `chromium_rebase_l10n`. but maybe this process was changed?